src/test/java/org/fundacionjala/coding/pablo/Movies/NewReleaseTest.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.coding.pablo.Movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Administrator on 3/23/2017.
+ */
+public class NewReleaseTest {
+
+    /**
+     * Test for calculate rented days less than 3.
+     */
+
+    @Test
+    public void calculateRentedDaysLessThan3() {
+
+        //given
+        NewRelease newRelease = new NewRelease("TestNewRelease");
```

Comment:
You should use the abstract class here.
```java
Movie newRelease = new NewRelease("TestNewRelease");
```

Please apply it in all your unit tests